CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 ## vNext
 
+### Features
```

Comment:
@daniloprates please be careful when adding items to the CHANGELOG so that they are added into the correct release.